cljfmt/src/cljfmt/core.cljc

```diff
@@ -306,11 +328,68 @@
       (cond-> (:insert-missing-whitespace? opts true)
         insert-missing-whitespace)
       (cond-> (:indentation? opts true)
-        (reindent (:indents opts default-indents)))
+        (reindent (:indents opts default-indents)
+                  (:alias-map opts {})))
       (cond-> (:remove-trailing-whitespace? opts true)
         remove-trailing-whitespace)))
 
+(defn- top-level-containing-form
```

Comment:
I think this could be written:

```clojure
(defn- top-level-form [zloc]
  (->> zloc
       (iterate z/up)
       (take-while (complement root?))
       last))
```

Without losing any information. As this isn't a public API, we don't need a docstring if the source code and function name makes the purpose of the function clear. To my mind, `(top-level-form zloc)` is self-explanatory.